apps/nextjs-website/react-components/components/PreFooter/PreFooter.tsx

```diff
@@ -71,60 +84,74 @@ const PreFooter = (props: PreFooterProps) => {
         >
           <Box sx={styles.main(isSmallScreen)}>
             <Typography
-              variant='h4'
+              variant='h6'
               color={theme === 'dark' ? 'white' : 'black'}
               mb={isSmallScreen ? 2 : 'unset'}
+              sx={{ fontSize: { xs: '18px', md: '24px' } }}
             >
               {title}
             </Typography>
-            {storeButtons?.hrefGoogle || storeButtons?.hrefApple ? (
+
+            {useStoreButtons ? (
```

Comment:
Remove the use of `useStoreButtons`.

The logic should go as follows:
if storeButtons exists, render the ones that exist.
else, if ctaButtons exists, render CtaButtons.